storage/pruning/triePruningStorer.go

```diff
@@ -12,7 +12,7 @@ import (
 const (
 	lastEpochIndex             = 1
 	currentEpochIndex          = 0
-	minNumOfActiveDBsNecessary = 2
+	minNumOfActiveDBsNecessary = 1
```

Comment:
why this change?

Reply:
Only 1 activeDB is needed for a complete snapshot. If it is set to 2, more DBs will remain open, even if it is not necessary.